scripts/run_bm.sh

```diff
@@ -40,7 +40,9 @@ BM3="com.arjuna.ats.jta.xa.performance.JTAStoreTests.*"
 
 function run_benchmarks {
   run_bm narayana/ArjunaCore/arjuna "$BM1" $1
-  run_bm narayana/ArjunaCore/arjuna "$BM2" $1
+  if [[ -z $2 ]] ; then
+    run_bm narayana/ArjunaCore/arjuna "$BM2" $1
```

Comment:
I suspect that the only time we call run_benchmarks is during a narayana build via the build script scripts/hudson/narayana.sh and only when the PERF_TESTS axis is enabled in CI.

Therefore I would just delete this line.

Then you will not need the PR on the main narayana repo and you won't need the other changes below (to pass in the extra arg).

Reply:
Okay, if run_benchmarks is only called during narayana build, then I will remove it.